contentcuration/contentcuration/frontend/channelEdit/vuex/clipboard/actions.js

```diff
@@ -89,76 +90,153 @@ export function loadClipboardNodes(context, { parent, ancestorId }) {
-            });
-          }
+        return context.dispatch('addClipboardNodes', {
+          nodes: clipboardNodes,
+          parent,
         });
-
-        context.commit('ADD_CLIPBOARD_NODES', clipboardNodes);
-        return clipboardNodes;
       });
     });
-  } else if (!parentNode || !isLegacyNode(parentNode)) {
-    // Has no child resources, and is either a new style clipboard node or not a clipboard node
-    // so fetch children of associated contentnode instead if it has any
-    const contentNode = context.getters.getClipboardNodeForRender(parent, ancestorId);
-    if (contentNode && contentNode.has_children) {
-      return context
-        .dispatch('contentNode/loadContentNodes', { parent: contentNode.id }, { root })
-        .then(contentNodes => {
-          // Be sure to put these in after the channels!
-          contentNodes.forEach(node => {
-            if (!(node.id in context.state.selected)) {
-              context.commit('UPDATE_SELECTION_STATE', {
-                id: node.id,
-                selectionState: SelectionFlags.NONE,
-                ancestorId,
-              });
-            }
-          });
-          return [];
+  }
+
+  // Since this must be a new clipboard node that's not the top-level ancestor, we'll need
+  // to find the real parent (which should already be loaded into contentNode Vuex state) and use
+  // it to load its real children. We also get the ancestor node, for uniquely referencing this
+  // copy of the source node in the clipboard
+  const contentNode = context.getters.getContentNodeForRender(parent);
+  const ancestor = context.getters.getClipboardAncestorNode(parent);
+  if (!ancestor) {
+    throw new Error('Unexpected null ancestor');
+  }
+
+  if (contentNode && contentNode.resource_count > 0) {
+    return context
+      .dispatch('contentNode/loadContentNodes', { parent: contentNode.id }, { root: true })
+      .then(nodes => {
+        // Ensure we don't add excluded nodes to our Vuex map
+        nodes = nodes
+          .filter(node => !isExcludedNode(ancestor, node))
+          .map(node => ({
+            // For new version clipboard nodes, we don't need `node.id` for anything
+            // so we can change it, and the clipboard should not be manipulating the
+            // source node anyway, normally requiring its ID. So we create a virtual
+            // clipboard node based off the source, since only its root ancestor
+            // exists in the literal clipboard tree
+            id: selectionId(node.id, ancestor.id),
+            kind: node.kind,
+            parent,
+            lft: node.lft,
+            source_node_id: node.node_id,
+            source_channel_id: node.channel_id,
+            total_count: node.total_count,
+            resource_count: node.resource_count,
+            // Be sure to set this flag for this as a new-versioned node
+            extra_fields: {
+              [ClipboardNodeFlag]: true,
+            },
```

Comment:
Without marking this virtual node, our vuex getter would say it's a legacy node.